learning/step_by_step/scripting.rst

```diff
@@ -20,14 +20,14 @@ tools that try to reduce the amount of code that needs to be written for
 specific tasks, to speed up development.
 
 In that sense, Godot has taken some useful design decisions towards that
-goal. The first and most important is the scene system. The aim of it is
-not obvious at first, but works well later on. That is, to relieve
-programmers from the responsibility of architecting code.
+goal. The first, and most important, is the scene system. The aim of it is
+not obvious at first, but is understandable later on. That is, it relieves
+programmers from the responsibility of designing and structuring code.
```

Comment:
"designing and structuring"

Just checking, does this change the meaning of the text accidentally? Programmers will still have to design and structure their classes even with Godot. I found the previous word "architecting" a little more appropriate here, since that implies more that Godot is helping you with the larger scale issue of overall code architecture, not the smaller scale local issues of individual class design and structure.